src/main/resources/org/jenkins/plugins/lockableresources/LockStep/help-inversePrecedence.html

```diff
@@ -0,0 +1,8 @@
+<div>
+	<p>
+		By default waiting builds get the lock in the same order the requested to acquire it.
+	</p>
+	<p>
+		By checking this option the newest build in the waiting queue will get the lock first.
```

Comment:
The build number is being used currently to pick up the newest, so it works for multiple builds of the same job but not for other jobs... need to fix it (probably using `run.getStartTimeInMillis()`). Thanks for finding it!